ui/admin/app/components/form/storage-bucket/aws/index.hbs

```diff
@@ -456,30 +456,41 @@
   {{/if}}
 
   {{! worker_filter }}
-  <Hds::Form::TextInput::Field
-    name='worker_filter'
-    @isRequired={{true}}
-    @value={{@model.worker_filter}}
-    @isInvalid={{@model.errors.worker_filter}}
-    disabled={{form.disabled}}
-    {{on 'input' (set-from-event @model 'worker_filter')}}
-    as |F|
-  >
-    <F.Label>{{t 'resources.storage-bucket.form.worker_filter.label'}}</F.Label>
-    <F.HelperText>
-      {{t 'resources.storage-bucket.form.worker_filter.help'}}
-      <Hds::Link::Inline @href={{doc-url 'storage-bucket.worker-filter'}}>
-        {{t 'actions.learn-more'}}
-      </Hds::Link::Inline>
-    </F.HelperText>
-    {{#if @model.errors.worker_filter}}
-      <F.Error data-test-worker-filter-error as |E|>
-        {{#each @model.errors.worker_filter as |error|}}
-          <E.Message>{{error.message}}</E.Message>
-        {{/each}}
-      </F.Error>
-    {{/if}}
-  </Hds::Form::TextInput::Field>
+  {{#let (unique-id) (unique-id) as |labelId helpId|}}
+    <Hds::Form::Fieldset
```

Comment:
Is this element going to repeat within the screen? or we will have just 1? I am wondering why we need a `unique-id` here.

My thinking:
I understand we need to define `aria-labelledby` and `aria-describedby` for a11y purposes, but if I am not mistaken, this same attributes are to provide a "human readable" id, while unique-id is more "computer" than "human" language.

Thoughts?

Reply:
This is true for `aria-label` but `aria-labeledby` and `aria-describedby` describe the _element_ that has the actual human readable text. In that case, it's perfectly fine to use a unique ID that's not very readable as that's now what is actually used by the assistive technology

Reply:
Oooh, so these are simply going to associate the label and help text to the fieldset but not the context used by the assistive technology?

Reply:
As per mdn, [seems about right what Zed mentions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby):
```
The aria-describedby attribute lists the [id](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id)s of the elements that describe the object. It is used to establish a relationship between widgets or groups and the text that describes them.
```
I am fine either way, but when using those, if there is no repetition of the elements, I personally prefer a human readable string than a generated uuid that changes by every render loop/refresh.

Reply:
> Oooh, so these are simply going to associate the label and help text to the fieldset but not the context used by the assistive technology?

It's the equivalent of a label's [for](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/for) in cases where you don't actually have a label element. 

> I am fine either way, but when using those, if there is no repetition of the elements, I personally prefer a human readable string than a generated uuid that changes by every render loop/refresh.

That's fair, I don't have a strong opinion on this so don't mind either way but just a static ID would also work in this case. However, I would keep the unique ID in the component as you don't exactly know how a consumer will use it.

Reply:
That is correct. Being a component, unique-id is the right way to go.